platforms/quorum/configuration/roles/create/constellation/tasks/main.yaml

```diff
@@ -25,7 +25,7 @@
 # This task adds a new node to the existing network
 - name: Adds a new node to the existing network
   shell: |
-    curl -X POST http://{{ geth_data.url }}:{{ geth_data.rpcport }} --data '{"jsonrpc":"2.0","method":"raft_addPeer","params": ["enode://{{ enode.enodeval }}@{{ enode.peer_name }}.{{ external_url }}:{{ enode.p2p_ambassador }}?discport=0&raftport={{ enode.raft_ambassador }}"], "id":{{ geth_data.nodeid }} }' --header "Content-Type: application/json"
+    curl -X POST http://{{ geth_data.url }}:{{ geth_data.rpcport }} --data '{"jsonrpc":"2.0","method":"raft_addPeer","params": ["enode://{{ enode.enodeval }}@{{ enode.peer_name }}.{{ enode.external_url }}:{{ enode.p2p_ambassador }}?discport=0&raftport={{ enode.raft_ambassador }}"], "id":{{ geth_data.nodeid }} }' --header "Content-Type: application/json"
```

Comment:
use uri ansible module  instead of shell

Reply:
Can this be addressed? @lakshyakumar